src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```
@@ -36,6 +43,11 @@
       <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
     </_MauiImplicitPackageReference>
   </ItemGroup>
+  <ItemGroup Condition=" '$(UseMaui)' == 'true' and '$(UseMauiCompat)' == 'true' ">
```

You probably don't even need to check `UseMaui`?
```suggestion
  <ItemGroup Condition=" '$(UseMauiCompat)' == 'true' ">
```

---

I didn't want someone to be able to put just UseMauiCompat and not have the rest of maui brought in, so just being extra cautious.